Xamarin.Forms.Platform.Android/Renderers/KeyboardExtensions.cs

```diff
@@ -31,9 +31,12 @@ public static InputTypes ToInputType(this Keyboard self)
 			else if (self is CustomKeyboard)
 			{
 				var custom = (CustomKeyboard)self;
-				bool capitalizedSentenceEnabled = (custom.Flags & KeyboardFlags.CapitalizeSentence) == KeyboardFlags.CapitalizeSentence;
-				bool spellcheckEnabled = (custom.Flags & KeyboardFlags.Spellcheck) == KeyboardFlags.Spellcheck;
-				bool suggestionsEnabled = (custom.Flags & KeyboardFlags.Suggestions) == KeyboardFlags.Suggestions;
+				bool capitalizedSentenceEnabled = custom.Flags.HasFlag(KeyboardFlags.CapitalizeSentence);
```

Comment:
We tend to avoid using `HasFlag` because has a substantial performance cost.

Reply:
Good to know. I noticed about 33 places in the code that used it so I just thought maybe it was personal preference.  I'll create a quick issue around that so the standard is established